00-introduction.adoc

```diff
@@ -1,199 +1,265 @@
 = Introduction
```

Comment:
General thought - since we're both going to be on the byline for this book, it might make sense to put together an introduction of who Phil is, and who Mike is, and why we started collaborating on this: you've been designing and building APIs for ages, and I've been building the experiences that consume them.

Otherwise this introduction does an odd switch between "I" and "we" between the first and second sections

Reply:
Yeah good spot, I was thinking of a foreword somewhere. Going to find a way for us to tag who each chapter is written later on.